corehq/apps/app_manager/views/releases.py

```diff
@@ -650,3 +665,39 @@ def toggle_build_profile(request, domain, build_id, build_profile_id):
                 build.build_profiles[build_profile_id].name
             ))
     return HttpResponseRedirect(reverse('download_index', args=[domain, build_id]))
+
+
+@require_deploy_apps
+def paginate_release_logs(request, domain, app_id):
+    limit = request.GET.get('limit')
+    page = int(request.GET.get('page', 1))
+    page = max(page, 1)
+    try:
+        limit = int(limit)
+    except (TypeError, ValueError):
+        limit = DEFAULT_PAGE_LIMIT
+
+    app_release_logs = ApplicationReleaseLog.objects.filter(app_id=app_id).order_by('-created_at')
+    paginator = Paginator(object_list=app_release_logs, per_page=limit)
+    current_page = paginator.get_page(page)
+
+    timezone = get_timezone_for_user(request.couch_user, domain)
+    transformed_logs = list(populate_data_app_release_logs(log, timezone) for log in current_page)
+
+    return JsonResponse({
+        'app_release_logs': transformed_logs,
+        'pagination': {
+            'total': paginator.count,
+            'num_pages': paginator.num_pages,
+            'current_page': page,
+            'more': page * limit < 10,
```

Comment:
This caught my eye because the calculation looked off (based on my guess of what "more" would be used for) but it looks like it isn't used at all. Should this be removed?